src/assets/i18n/en.json

```diff
@@ -87,7 +87,7 @@
 
 		"headline1": "Gemeinsam machen wir",
 		"headline2": "Kompetenzerwerb sichtbar",
-		"text1": "Badges sind digitale Nachweise, die wie Zertifikate einfach und zeiteffizient vergeben und gesammelt werden können, um Lernerfolge sichtbar zu machen – sowohl für digitale Lernangebote als auch für Angebote vor Ort.",
+		"text1": "Badges are digital evidence that, like certificates, can be awarded and collected easily and time-efficiently in order to make learning successes <span class='red-text'> and skills acquisition </span> visible - both for digital learning offers and for on-site offers.",
```

Comment:
The text is not supposed to be red. It was just to mark the changes in figma.

Reply:
I see.